### PR TITLE
Update to Cadence v0.37.0

### PR DESCRIPTION
**Pull request by turbolent**
## Description

Automatically update to:
- [onflow/flow-go-sdk v0.37.0](https://github.com/onflow/flow-go-sdk/releases/tag/v0.37.0)
- [onflow/cadence v0.37.0](https://github.com/onflow/cadence/releases/tag/v0.37.0)

### CODE REVIEW

**Comment by turbolent** [comment]
So it appears `flow-go-sdk` was update to a new `flow-go/crypto` version, but `flow-go` itself was not yet 🤔

**Review by janezpodhostnik — APPROVED** [review]

File: network/p2p/translator/unstaked_translator_test.go [hunk 1]
```diff
@@ -99,7 +100,7 @@ func createPeerIDFromAlgo(t *testing.T, sa fcrypto.SigningAlgorithm) peer.ID {
 }
 
 func createSeed(t *testing.T) []byte {
-	seedLen := int(math.Max(fcrypto.KeyGenSeedMinLenECDSAP256, fcrypto.KeyGenSeedMinLenECDSASecp256k1))
+	seedLen := int(math.Max(fcrypto.KeyGenSeedMinLen, fcrypto.KeyGenSeedMinLen))
```
Comment (tarakby) [inline network/p2p/translator/unstaked_translator_test.go:103]:
```suggestion
	seedLen := fcrypto.KeyGenSeedMinLen
```

**Review by tarakby — APPROVED** [review]
Thank you for making the changes 🙏🏼

File: consensus/hotstuff/timeoutcollector/aggregation_test.go [hunk 2]
```diff
@@ -7,11 +7,12 @@ import (
 
 	"github.com/stretchr/testify/require"
 
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/crypto/hash"
+
```
[the remark this answers is not in the document]
Comment (turbolent) [reply consensus/hotstuff/timeoutcollector/aggregation_test.go:12]: Go fmt organizes/groups the imports

**Comment by turbolent** [comment]
bors merge

**Comment by turbolent** [comment]
bors retry

**Pull request merged**